### PR TITLE
Allow 'IsWpfTempProject' to be set to prevent duplicate imports when restoring packages in temporary projects

### PR DESCRIPTION
**Pull request by ryalanms**
WPF for .NET Core is adding a restore step to temporary project compilation for PackageReferences.  The existing Arcade workaround needs to be temporarily updated to prevent duplicate imports, then removed entirely after the next WPF Preview release.

### CODE REVIEW

**Comment by markwilkie** [comment]
Please be sure there's a tracking issue to follow up after the next WPF preview release.

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.props [hunk 1]
```diff
@@ -7,8 +7,8 @@
     Since .NET Framework 4.7.2 WPF temp project name ends with _wpftmp suffix and keeps the language specific extension (e.g. csproj). 
   -->
   <PropertyGroup>
-    <IsWpfTempProject>false</IsWpfTempProject>
-    <IsWpfTempProject Condition="$(MSBuildProjectName.EndsWith('_wpftmp'))">true</IsWpfTempProject>
+    <IsWpfTempProject Condition="'$(IsWpfTempProject)' == '' and $(MSBuildProjectName.EndsWith('_wpftmp'))">true</IsWpfTempProject>
```
Comment (rladuca) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.props]: Why not just put the condition on the PropertyGroup and leave the ordering as-is?

**Comment by ryalanms** [comment]
@markwilkie, I've opened this issue to track the removal: https://github.com/dotnet/arcade/issues/3129.  Thanks.

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.props [hunk 2]
```diff
@@ -6,8 +6,8 @@
     Determine whether the project is WPF temp project.
     Since .NET Framework 4.7.2 WPF temp project name ends with _wpftmp suffix and keeps the language specific extension (e.g. csproj). 
   -->
-  <PropertyGroup>
-    <IsWpfTempProject>false</IsWpfTempProject>
+  <PropertyGroup Condition="'$(IsWpfTempProject)' == ''">
+    <IsWpfTempProject >false</IsWpfTempProject>
```
Comment (rladuca) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.props]: nit: space

**Pull request merged**